cmake/projects/OpenSSL/hunter.cmake

```diff
@@ -65,6 +65,95 @@ hunter_add_version(
     df0ee4811c87c209ebadb4e6b203d1e560d00f9a
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    OpenSSL
+    VERSION
+    "1.1.1g"
+    URL
+    "https://github.com/openssl/openssl/archive/OpenSSL_1_1_1g.tar.gz"
+    SHA1
+    33324ff957edaae8ae575817b456320378da46ff
+)
+
+# NB: Only use this version if you need arm64 support before official release!
+hunter_add_version(
+    PACKAGE_NAME
+    OpenSSL
+    VERSION
+    "1.1.1g-p0"
+    URL
+    "https://github.com/cpp-pm/openssl/archive/v1.1.1g-p0.tar.gz"
+    SHA1
+    fb3bc6467d37f7536f5ff7a1ea25552cee899275
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    OpenSSL
+    VERSION
+    "1.1.1h"
+    URL
+    "https://github.com/openssl/openssl/archive/OpenSSL_1_1_1h.tar.gz"
+    SHA1
+    e581875cc63a29a2e9ae6924f5e65fd79dd3fc2e
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    OpenSSL
+    VERSION
+    "1.1.1i"
+    URL
+    "https://github.com/openssl/openssl/archive/OpenSSL_1_1_1i.tar.gz"
+    SHA1
+    b1fcae20e23561f3ad02c61c80ba10d8790e3034
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    OpenSSL
+    VERSION
+    "1.1.1j"
+    URL
+    "https://github.com/openssl/openssl/archive/OpenSSL_1_1_1j.tar.gz"
+    SHA1
+    327a400b8bb58058604f83c7e839fea11bba095e
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    OpenSSL
+    VERSION
+    "1.1.1j"
+    URL
+    "https://github.com/openssl/openssl/archive/OpenSSL_1_1_1j.tar.gz"
+    SHA1
+    327a400b8bb58058604f83c7e839fea11bba095e
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    OpenSSL
+    VERSION
+    "1.1.1k"
+    URL
+    "https://github.com/openssl/openssl/archive/OpenSSL_1_1_1k.tar.gz"
+    SHA1
+    95a5b6a0f9e866b1b131d71ec13b7f2b83a93da1
+)
+
+hunter_add_version(
```

Comment:
Perhaps, it is better to leave really used version

Reply:
Wasn't OpenSSL update one of our decisions?

Reply:
It about configs of others versions.